lib/langchain/conversation.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 module Langchain
-  class Chat
+  class Conversation
```

Comment:
Think we can also annotate the code, write out what it's used for, what it does and usage examples?

Reply:
Adding in the next PR https://github.com/andreibondarev/langchainrb/pull/169